File: src/test/java/eotree/data/TestEOIntData.java
package eotree.data;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

/**
 * EO int data tests.
 */
public class TestEOIntData {

    @Test
    public void testGenerateEOZeroIndent() {
        var f = new EOIntData(42);

        assertEquals(f.generateEO(0), "42");
    }

    @Test
    public void testGenerateEONonZeroIndent() {
        var f = new EOIntData(42);

        assertEquals(f.generateEO(1), "  42");
    }
}

File: src/test/kotlin/eotree/data/TestEOIntData.kt
package eotree.data

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import org.polystat.j2eo.eotree.data.EOIntData

/**
 * EO int data tests.
 */
class TestEOIntData {
    @Test
    fun testGenerateEOZeroIndent() {
        val f = EOIntData(42)
        Assertions.assertEquals(f.generateEO(0), "42")
    }

    @Test
    fun testGenerateEONonZeroIndent() {
        val f = EOIntData(42)
        Assertions.assertEquals(f.generateEO(1), "  42")
    }
}
